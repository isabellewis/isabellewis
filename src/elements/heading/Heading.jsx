import React from 'react';
import styled from 'styled-components';

export const HeadingOne = styled.h1`
	font-size: 200px;
	margin: 40px 20px;
	overflow-wrap: break-word;
	word-break: break-all;
`;

export const Heading = ({
	title,
}) => (
	<HeadingOne className="heading-text">
		{ title }
	</HeadingOne>
);
