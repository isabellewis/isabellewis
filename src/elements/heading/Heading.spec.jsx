import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { Heading } from './Heading';

describe('Heading', () => {
	let component;

	beforeEach(() => {
		component = shallow(<Heading title='Heading test text' />);
	});

	it('should match the snapshot', () => {
		expect(component).toMatchSnapshot();
	});
});