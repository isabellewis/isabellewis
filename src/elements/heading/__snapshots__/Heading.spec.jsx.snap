// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Heading
    title="Heading test text"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": "Heading test text",
      "className": "heading-text",
    },
    "ref": null,
    "rendered": "Heading test text",
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "Heading test text",
        "className": "heading-text",
      },
      "ref": null,
      "rendered": "Heading test text",
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
